src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/controller/PersonControllerIntTest.kt

```diff
@@ -61,24 +61,6 @@ class PersonControllerIntTest() : IntegrationTestBase() {
     )
   }
 
-  @Test
-  fun `should return HTTP Location header containing the URL of new person`() {
-    // Given
-    val personJson = objectMapper.writeValueAsString(minimumPerson)
-
-    // When
-    val result = mockMvc.perform(
-      post("/person")
-        .contentType(MediaType.APPLICATION_JSON)
-        .content(personJson)
-        .headers(setAuthorisation(roles = listOf(VIEW_PERSON_DATA_ROLE))),
-    )
-      .andReturn()
-
-    // Then
-    // TODO
-  }
-
```

Comment:
I have deleted this as it has no assertions. We are some way away from providing this endpoint to other teams.